disco_aws_automation/disco_deploy.py

```diff
@@ -436,8 +436,30 @@ def handle_blue_green_ami(self, pipeline_dict, ami, old_group,
         new_group_config["min_size"] = min_size
         new_group_config["max_size"] = max_size
 
-        # Spinup our new autoscaling group in testing mode, making one even if one already exists.
-        self._disco_aws.spinup([new_group_config], create_if_exists=True, testing=True)
+        try:
+            # Spinup our new autoscaling group in testing mode, making one even if one already exists.
+            self._disco_aws.spinup([new_group_config], create_if_exists=True, testing=True)
+        except Exception:
```

Comment:
No, but are there any exceptions that could be thrown where we wouldn't want to tear down the new ASG and its ELB?